### ISSUE TITLE
Tutorial for Helm-based Operators

### ISSUE THREAD

**Issue by olaf-meyer** [operator-framework/operator-sdk#4469]
The help text on how to use the plugin helm is very short and unspecific. It would be great show show examples how `operator-sdk init --plugins=helm` can be used.

Here is the output:
``` bash
operator-sdk init --plugins=helm --help
Initialize a new project.

For further help about a specific project version, set --project-version.

Usage:
  operator-sdk init [flags]

Examples:
  # Help for initializing a project with version "2"
  operator-sdk init --project-version="2" -h

  # Help for initializing a project with version "3-alpha"
  operator-sdk init --project-version="3-alpha" -h

Flags:
  -h, --help                     help for init
      --plugins strings          Name and optionally version of the plugin to initialize the project with. Available plugins: ("ansible.sdk.operatorframework.io/v1", "go.kubebuilder.io/v2", "go.kubebuilder.io/v3", "helm.sdk.operatorframework.io/v1")
      --project-version string   project version, possible values: ("2", "3-alpha") (default "3-alpha")

Global Flags:
      --verbose   Enable verbose logging
```
Thank you much

**Comment by estroz** [comment]
@olaf-meyer this is coming once #4402 is merged, and is a duplicate of #3798.
